### ISSUE TITLE
Better description on error during plugin update

### ISSUE THREAD

**Issue by K232** [getgrav/grav-plugin-admin#80]
I'm not sure whether it's a problem of Grav Admin Panel or [Archive Plus Plugin](https://github.com/Sommerregen/grav-plugin-archive-plus) from @Sommerregen . I've posted it here because I'm not sure what the error message means so a better description given by Admin Panel to the user would be helpful.

I've tried to update Archive Plus Plugin from 1.3 to 1.3.1 via Admin Panel. I get the following error message:

*rename(D:/mypath/wwwroot/cache//tmp/Grav-55d42de14b827/Sommerregen-grav-plugin-archive-plus-c254496/,D:/mypath/wwwroot/user/plugins/archive_plus): The specified path is invalid. (code: 161)*

All other plugins (e.g. Admin Panel 0.3.0) and grav itself are on latest version (another thing: Maybe display version of Grav on Dashboard?). Now I've seen that Archive Plus has been removed automatically, probably due to the problem with rename.

**Comment by Sommerregen** [comment]
Hi @K232 ,

since this is maybe connected with my plugin, I will join in. I also noticed such a strange behaviour on my dev machine. I got an internal error message, but haven't had time to track it down. The changes I made from **Archive Plus v1.3.0** to **Archive Plus v1.3.1** can be seen here https://github.com/Sommerregen/grav-plugin-archive-plus/compare/v1.3.0...master (in my opinion nothing really special or that cause such an error). Maybe I'm wrong...

According to the documentation of your above error code _The specified path is invalid. (code: 161)_ it must have to do with the **Admin Panel**, since the error reads

```
This is because the target path, where the file should be moved to, contains a duplicate /.
```

This is indeed the case here. Look at your first path. Before "tmp" there are **two slashes** (which are not allowed)! I'm sure @rhukster or @flaviocopes know the correct line(s) to fix this issue either in admin panel or in Grav. :smile:
